src/Microsoft.PowerShell.Commands.Management/commands/management/Process.cs

```diff
@@ -1550,7 +1559,10 @@ private void AttachDebuggerToProcess(Process process)
                     catch (CimException e)
                     {
                         string message = e.Message;
-                        if (!string.IsNullOrEmpty(message)) { message = message.Trim(); }
+                        if (!string.IsNullOrEmpty(message))
+                        {
+                            message = message.Trim();
+                        }
```

Comment:
```suggestion
                        message = message?.Trim();
```

Reply:
@iSazonov I think in this case `Message` cannot be null, since its the exception message and would only return empty string. 

It has `The error message that explains the reason for the exception, or an empty string ("")` in the return description.

Reply:
Actually I think its fine, will just do `string message = e.Message?.Trim();`.